### PR TITLE
fix(files): move focus to sidebar on open

### LINKED ISSUE #41877: [BITV] 9.2.4.3/3.3 - When opening the "Details" section through the "Actions" menu in the table, the keyboard focus is not shifted to the details view; instead, it is moved to the beginning of the next row of the table. (1)
https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-89806fe784-v3-n3

### PR DESCRIPTION
**Pull request by ShGKme**
* Resolves: #41877

## Summary

See commits for details.

See https://github.com/nextcloud-libraries/nextcloud-vue/pull/5219 for screenshots

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by JuliaKirschenheuter — APPROVED** [review]
Works great too! Thank you!

File: apps/files/src/views/Sidebar.vue [hunk 1]
```diff
@@ -471,6 +470,10 @@ export default {
 				throw new Error(`Invalid path '${path}'`)
 			}
 
+			// Only focus the tab, when the current file/tab in changed in aleady opened sidebar
```
Comment (emoral435) [inline apps/files/src/views/Sidebar.vue:473]:
```suggestion
			// Only focus the tab when the current file/tab selected is already opened in the sidebar
```

**Review by emoral435 — APPROVED** [review]
Outside of a minor nitpick, LGTM! 🐘💙

**Comment by ShGKme** [comment]
> /compile /

This PR require `@nextcloud/vue` update first

File: apps/files/src/views/Sidebar.vue (hunk 1 above)
Comment (ShGKme) [reply apps/files/src/views/Sidebar.vue]: I fixed some typos:
```
// Only focus the tab when the selected file/tab is changed in already opened sidebar
```

But kept the old meaning. It is used not when this file/tab was selected, but when any file/tab was selected and the sidebar was opened.

Comment (emoral435) [reply apps/files/src/views/Sidebar.vue]: Ahhh - gotcha! 💙

File: apps/files/src/components/FileEntry/FileEntryActions.vue [hunk 2]
```diff
@@ -53,6 +53,7 @@
 				:close-after-click="!isMenu(action.id)"
 				:data-cy-files-list-row-action="action.id"
 				:is-menu="isMenu(action.id)"
+				:aria-label="mountType === 'shared' && action.id === 'sharing-status' ? action.title([source]) : actionDisplayName(action)"
```
Comment (skjnldsv) [inline apps/files/src/components/FileEntry/FileEntryActions.vue:55]: That seems really specific, where is this coming from and what does it have to do with that PR? :)

**Review by skjnldsv — CHANGES_REQUESTED** [review]
See comment

File: apps/files/src/components/FileEntry/FileEntryActions.vue (hunk 2 above)
Comment (ShGKme) [reply apps/files/src/components/FileEntry/FileEntryActions.vue:55]: From the next one line (`title` prop) and the content of the button 7 lines below. I can move it to a new PR, but a button must have a label or text.

**Comment by ShGKme** [comment]
- Rebased onto `master` with required `@nextcloud/vue` changes
- Fixing labeling of inline files actions is moved to another PR

File: apps/files/src/components/FileEntry/FileEntryActions.vue (hunk 2 above)
Comment (ShGKme) [reply apps/files/src/components/FileEntry/FileEntryActions.vue]: @skjnldsv moved to another PR

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**